azuredevops/provider.go

```diff
@@ -141,6 +167,134 @@ func Provider() *schema.Provider {
 				Description: "The personal access token which should be used.",
 				Sensitive:   true,
 			},
+			"sp_client_id": {
```

Comment:
@davidcorrigan714 Apologies if this is already documented somewhere, but I'm wondering why the choice was made to use different env vars to the `azurerm`, `azuread` and `azapi` providers? If it worked the same as those providers by default, it might reduce the setup complexity for some users. E.g. `ARM_USE_OIDC`, etc.

Reply:
I can go double check this, the main concern is ensuring that they can be configured independently when needed and support multiple instances to different accounts. Not sure the Azure one suffered that bug but the AWS implementation had issues with that, I should find that bug and double check what they did there. The environment variable names for them do seem to provider specific though, like this isn't Azure Resource manager so ARM_.... doesn't really make sense to me

Reply:
Here's a workaround in case someone else is stumbling onto this:
```terraform
provider "azuredevops" {
  // … other settings

  // prevent enabling OIDC when `ARM_*` vars are detected in CI
  use_oidc  = false
  // not used, but these need to be set to valid GUID if `use_oidc` is set
  client_id = "deadbeef-dead-d34d-dead-deadbeefdead"
  tenant_id = "deadbeef-dead-d34d-dead-deadbeefdead"
}
```